tests/recipient_validation/test_phone_number.py

```diff
@@ -391,3 +391,155 @@ def test_try_format_recipient_doesnt_throw():
 
 def test_format_phone_number_human_readable_doenst_throw():
     assert format_phone_number_human_readable("ALPHANUM3R1C") == "ALPHANUM3R1C"
+
+
+class TestPhoneNumbeClass:
```

Comment:
typo? Shouldn't this be `class TestPhoneNumberClass`?

Reply:
Fixed